playbooks/common/openshift-cluster/initialize_facts.yml

```diff
@@ -6,12 +6,140 @@
 
 - name: Initialize host facts
   hosts: oo_all_hosts
-  roles:
-  - openshift_facts
   tasks:
-  - openshift_facts:
+  - name: load openshift_facts module
+    include_role:
+      name: openshift_facts
+
+  # TODO: Should this role be refactored into health_checks??
+  - name: Run openshift_sanitize_inventory to set variables
+    include_role:
```

Comment:
@ingvagabund, `include_role` is the future as per our very own cluster lifecycle meeting with the ansible consultant.